helm_charts_updater/readme_generator.py

```
@@ -35,34 +72,84 @@ def _generate_table(charts: list) -> PrettyTable:
-    def _replace_table(self, table: PrettyTable):
+    def _replace_table(self, table: PrettyTable) -> None:
+        """Replace the table content between markers in the README.
+
+        Args:
+            table: The PrettyTable to insert into the README.
+
+        Raises:
+            IndexError: If start or end marker is not found in the README,
+                or if end marker appears before start marker.
+        """
         logging.info("Replacing table...")
```

Ruff (0.14.13)

`info()` call on root logger

(LOG015)